### PR TITLE
DAOS-10600 rpm: Update UCX RPM to 1.12

### PR DESCRIPTION
**Pull request by kjacque**
Updated the specfile to pull and build UCX v1.12. Based the update
on the CentOS8 version of the UCX v1.11 specfile.

Signed-off-by: Kris Jacque <kristin.jacque@intel.com>

### CODE REVIEW

**Comment by kjacque** [comment]
Tested this on Rocky8.5, was able to build and install the RPMs, and DAOS interacted with them correctly.

Haven't tested on EL7 or SUSE/Leap yet.

**Comment by brianjmurrell** [comment]
You will want to add `el8` to the list of distros in `Jenkinsfile` and perhaps remove `leap15` until you are ready to try porting the specfile to leap15.

**Comment by kjacque** [comment]
With some small tweaks, I've got the RPM building on Leap15 and CentOS7.9.

File: ucx.spec [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+%description vfs
+Provides a virtual filesystem over FUSE which allows real-time monitoring of UCX
+library internals, protocol objects, transports status, and more.
+
+%files vfs
+%{_libdir}/ucx/libucs_fuse.so.*
+%{_bindir}/ucx_vfs
+%endif
+
+%changelog
```
Comment (brianjmurrell) [inline ucx.spec:315]: Line 23 says the version is `Version: 1.12.1`.  Do you need an entry in this `%changelog`?

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Try removing https://github.com/daos-stack/openucx/blob/c4a7743b08ae42266bcdfe1f9246e7f0e3c2be7e/packaging/Makefile_packaging.mk#L101-L105 and see if that fixes the Test stage.

File: ucx.spec (hunk 1 above)
Comment (kjacque) [reply ucx.spec:315]: Added he changelog. Your script change let it pass the test stage, too. Thanks!

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
Looks good.  Can I ask you to rebase the `packaging/` subdir on `bmurrell/remove-repo-install` as the formal fix for that problem to ensure that it all still passes here and then I can get that landed?  Much thanks.

**Review by brianjmurrell — APPROVED** [review]

**Comment by brianjmurrell** [comment]
@kjacque requested landing in Skype.

**Pull request merged**